src/components/Button.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const Button = props => {
+  const widthC = {
+    fifty: '50%',
+    half: '25%',
+  };
+  let widthB;
+  /* eslint-disable */
```

Comment:
Let's try to remove ESLint disabling rules comments and fix the code accordingly.